src/main/java/it/pagopa/transactions/utils/MailConfidentialDataUtility.java

```diff
@@ -0,0 +1,55 @@
+package it.pagopa.transactions.utils;
+
+import it.pagopa.ecommerce.commons.domain.Confidential;
+import it.pagopa.ecommerce.commons.domain.v1.Email;
+import it.pagopa.ecommerce.commons.utils.ConfidentialDataManager;
+import it.pagopa.transactions.exceptions.ConfidentialDataException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+@Component
+@Slf4j
+public class MailConfidentialDataUtility {
```

Comment:
For consistency

```suggestion
public class MailConfidentialDataUtils {
```

But personally I like

```suggestion
public class ConfidentialDataMailUtils {
```

or 

```suggestion
public class ConfidentialMailUtils {
```

even more :)

Reply:
Fixed into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/150/commits/c0ff765bde5bc8d25a69ff0d7e92d77dae01ef6f